extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/customexceptions/FirstResource.java

```diff
@@ -36,6 +36,24 @@ public String throwsVariousExceptions(@RestQuery String name) {
         throw new RuntimeException();
     }
 
+    @GET
+    @Path("uni")
+    @Produces("text/plain")
+    public Uni<String> uni(@RestQuery String name) {
+        if (name.startsWith("IllegalArgument")) {
+            throw new IllegalArgumentException();
```

Comment:
Actually, this is not the second problem: the second problem was returning a Uni failure, so with these exceptions inside the Uni, not thrown in the execution of the endpoint.
Perhaps your fix fixes that but the test doesn't test it.

Reply:
Ah yeah. I was too quick to update the test

Reply:
Fixed